src/client/app/offers/edit_offer.html

```diff
@@ -0,0 +1,130 @@
+<!-- New offer, second step, register -->
+<section>
+  <div class="container edit-offer">
+    <div>
+      <h2 class="text-left">Редагування пропозиції</h2>
+    </div>
+    <div class="form-group row">
+      <div class="col-sm-10">
+        <form editable-form name="editableForm" onaftersave="vm.saveUser()">
+          <div class="form-group row">
+            <label class="control-label col-sm-2">Назва</label>
+            <div class="col-sm-10">
+              <div>
+                <span editable-text="vm.user.offer" e-name="name" onbeforesave="checkName($data)" e-required ng-model="vm.user.offer">{{ vm.user.offer || 'this one teststing' }}</span>
+              </div>
+            </div>
+          </div>
+          <div>
+            <ol class="breadcrumb">
+              <li>
+                <a href="#">Одяг</a>
+              </li>
+              <li>
+                <a href="#">Дитячий одяг</a>
+              </li>
+              <li>Светри</li>
+            </ol>
+          </div>
+          <hr>
+          <div class="form-group row">
+            <label class="control-label col-sm-2">Опис потреби</label>
+            <div class="col-sm-10">
+              <div>
+                <a href="#" editable-textarea="vm.user.offerText" e-rows="7" e-cols="90" ng-model="vm.user.offerText">
+                  <pre>{{ vm.user.offerText || 'no description' }}</pre>
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label class="control-label col-sm-2">Фото:</label>
+            <div class="col-sm-10">
+              <input type="file" id="exampleInputFile">
+            </div>
+          </div>
+          <div class="form-group row">
+            <label class="control-label col-sm-2">Виберіть розмір:</label>
+            <div class="col-sm-10">
+              <span editable-select="vm.user.size" e-name="name" e-ng-options="s.value as s.text for s in vm.size" ng-class="form-control" ng-model="vm.user.size">
+              {{ vm.user.size || 'XL' }}
```

Comment:
the best practice (mentioned in the styleguide) is to initialize variables in the controller.
`||` for setting default values can behave in different ways.